lib/dnsimple/client/zones.rb

```diff
@@ -77,6 +77,23 @@ def zone(account_id, zone_id, options = {})
         Dnsimple::Response.new(response, Struct::Zone.new(response["data"]))
       end
 
+      # Gets a zone file from the account.
+      #
+      # @see https://developer.dnsimple.com/v2/zones/#get-file
+      #
+      # @param  [Fixnum] account_id the account ID
+      # @param  [#to_s] zone_id the zone name
```

Comment:
In Go we used `zone_name`. I think it's more appropriate, as this is actually the name, not the ID.